fbc-integration-test/catalog-source/release/dev-workspace/release-plan.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleasePlan
+metadata:
+  labels:
+    release.appstudio.openshift.io/auto-release: 'true'
+    release.appstudio.openshift.io/standing-attribution: 'true'
+  name: manual-fbc-release
```

Comment:
why manual? we are using standing-attribution and auto-release. Seems like automated